base/model/src/main/java/org/eclipse/ditto/base/model/headers/DittoHeaderDefinition.java

```diff
@@ -221,6 +221,45 @@ public enum DittoHeaderDefinition implements HeaderDefinition {
      */
     REPLY_TARGET("ditto-reply-target", Integer.class, false, false, HeaderValueValidators.getIntValidator()),
 
+    /**
+     * Header mapping key for the target connection ID to divert responses to.
+     * When configured in a source's headerMapping, responses from that source will be diverted
+     * to the specified connection instead of being sent to the configured replyTarget.
+     * The value should be a valid connection ID string or header placeholder.
+     *
+     * @since 3.8.0
+     */
+    DITTO_DIVERT_RESPONSE_TO("divert-response-to",
+            String.class,
+            true,
+            false,
+            HeaderValueValidators.getNonEmptyValidator()),
+
+    /**
+     * Header mapping key for specifying which response types should be diverted.
+     * Value should be a comma-separated list of response types: "response,error,nack".
+     * If not specified, all response types will be diverted.
+     *
+     * @since 3.8.0
+     */
+    DITTO_DIVERT_EXPECTED_RESPONSE_TYPES("divert-expected-response-types",
```

Comment:
Only Ditto internal enum values start with `DITTO_` - so this one should not, because it is marked as `readFromExternalHeaders`